modules/remotecontrol/Receiver.js

```diff
@@ -0,0 +1,47 @@
+/* global APP, JitsiMeetJS */
+const ConferenceEvents = JitsiMeetJS.events.conference;
+
+/**
+ * This class represents the receiver party for a remote controller session.
+ * It handles "remote-control-event" events and sends them to the
+ * API module. From there the events can be received from wrapper application
+ * and executed.
+ */
+class Receiver {
+    /**
+     * Creates new instance.
+     * @constructor
+     */
+    constructor() {}
+
+    /**
+     * Attaches listener for ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED events.
+     */
+    start() {
+        APP.conference.addConferenceListener(
+            ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
+            this._onRemoteControlEvent);
+    }
+
+    /**
+     * Removes the listener for ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED
+     * events.
+     */
+    stop() {
+        APP.conference.removeConferenceListener(
+            ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
+            this._onRemoteControlEvent);
+    }
+
+    /**
+     * Sends "remote-control-event" events to to the API module.
+     * @param {JitsiParticipant} participant the controller participant
+     * @param {Object} event the remote control event.
```

Comment:
"event" is not "the remote control event", but it's a structure received over a DataChannel. event.event is expected to be the remote control event, so the comment is slightly misleading.